app/views/enlistments/index.xml.builder

```diff
@@ -4,7 +4,9 @@ xml.response do
   xml.items_returned @enlistments.length
   xml.items_available @enlistments.total_entries
   xml.first_item_position @enlistments.offset
-  xml.result do
-    xml << render(partial: 'enlistment', collection: @enlistments, locals: { builder: xml })
+  if @enlistments.present?
+    xml.result do
+      xml << render(partial: 'enlistment', collection: @enlistments, locals: { builder: xml })
+    end
   end
```

Comment:
Should there be a valid XML response when `@enlistments.present?` is a falsy value?

Reply:
We should return a valid responsę, as it is a valid request without any error. 
In this case, when `@enlistments.present?` is a falsy value, the response will not contain `result` element.
```
<?xml version="1.0" encoding="UTF-8"?>
<response>
    <status>success</status>
    <items_returned>0</items_returned>
    <items_available>0</items_available>
    <first_item_position>0</first_item_position>
</response>
```

Reply:
Perfect; thank you!